rir/tests/pir_check.R

```diff
@@ -9,10 +9,10 @@ stopifnot(pir.check(
   f <- function(){
     j <- 0
     while (j < 2) {
-    c(j)
+    vector("integer",0)
```

Comment:
why doesn't it work with `c`? this should be a builtin safe on non-objects, not?

Reply:
`c` uses ldFun